algorithms.go

```diff
@@ -104,8 +111,17 @@ func leakyBucket(c cache.Cache, r *RateLimitReq) (*RateLimitResp, error) {
 			return tokenBucket(c, r)
 		}
 
-		rate := b.Duration / r.Limit
+		duration := b.Duration
+		if r.Behavior == Behavior_DURATION_IS_GREGORIAN {
+			n := time.Now()
```

Comment:
Just for my understanding `time.Now()` will return the local time of where the service is running?

Reply:
Yes, in our production environments the localtime zone is set to UTC. So this will be UTC. Other server/container deployments might want to use their localtime zone.